Source/SIMPLib/DataArrays/NeighborList.hpp

```diff
@@ -792,30 +792,41 @@ class NeighborList : public IDataArray
     {
       QString info;
       QTextStream ss (&info);
-      if(format == SIMPL::HtmlFormat)
+
+      QString bgColor = "";
+      QString nameBgColor = "";
+
+      switch(format)
+      {
+      case SIMPL::ToolTipFormat:
+        bgColor = "bgcolor=\"#FFFCEA\"";
+        nameBgColor = "bgcolor=\"#E9E7D6\"";
+      case SIMPL::HtmlFormat:
```

Comment:
If fall-through is intentional, make sure to document it ever time.  `// [[falthrough]]` should be good enough while remaining short and easy to read.  We can't use the fallthrough attribute as it's C++17, but it's everything we would want to use while remaining far more easily readable than a longer comment stating that we want to use `[[fallthrough]]`